src/OpenLoco/Audio/Audio.h

```diff
@@ -92,16 +92,12 @@ namespace OpenLoco::Audio
     void unpauseSound();
     void playSound(Vehicles::Vehicle2or6* t);
     void playSound(SoundId id, const Map::Pos3& loc);
+    // FOR HOOKS ONLY DO NOT USE FOR OPENLOCO CODE
     void playSound(SoundId id, const Map::Pos3& loc, int32_t pan);
     void playSound(SoundId id, int32_t pan);
     void playSound(SoundId id, const Map::Pos3& loc, int32_t volume, int32_t frequency);
```

Comment:
Perhaps we can use an attribute that IDEs would pick up on, e.g.:
```suggestion
    [[deprecated]] void playSound(SoundId id, const Map::Pos3& loc, int32_t pan);
    [[deprecated]] void playSound(SoundId id, int32_t pan);
    [[deprecated]] void playSound(SoundId id, const Map::Pos3& loc, int32_t volume, int32_t frequency);
```

Reply:
(only the first one shouldn't be used by anything new) yeah thats an idea but at the same time would need to add some warning disables in hooks.

Reply:
Good point. We already set several compiler flags for Hooks.cpp, so that should be easy enough to extend, I think.

Reply:
Either way im not going to break the build just now to add the warnings it can be addressed by the issue that cleans up the users of this.

Reply:
> (only the first one shouldn't be used by anything new)

Could you clarify this? It wasn't obvious to me. Not too bothered about using attributes; I can see it might cause a bit more trouble than it's worth.